dashboard/test/controllers/maker_controller_test.rb

```diff
@@ -9,6 +9,43 @@ class MakerControllerTest < ActionController::TestCase
     @school = create :school
   end
 
+  test "home redirects to sign-in when user is signed out" do
+    assert_queries 0 do
+      get :home
+    end
+
+    assert_redirected_to '/users/sign_in'
+  end
+
+  test "home loads for student" do
+    # Fake CSD6 script for progress info
+    csd6_script = create :script, name: Script::CSD6_NAME
+    create :script_level, script: csd6_script
+    student = create :student
```

Comment:
Why not create the student in the setup step?

Reply:
Only because we're not reusing it anywhere else right now.  It might make sense to move it there for consistency.